docs/playbooks/integrations/lit.md

```diff
@@ -0,0 +1,730 @@
+---
```

Comment:
this is the bulk of the new stuff. Lit has a bunch of features, and I'm not quite sure how exactly it works. but, if you try to read data from [lit_encrypt_11155111_1314](https://testnets.tableland.network/api/v1/query?statement=SELECT%20*%20from%20lit_encrypt_11155111_1314), you can see what the raw/encrypted data looks like. 

you basically set up an access control condition, which can be _any_ arbitrary code (e.g., JS script). they have built-in methods for reading EVM contracts, which is what i do below. i.e., my access control calls the Tableland registry contract's `ownerOf` method and requires the one trying to decrypt is the owner of the table. this condition is used when writing data to the table as well as when decrypting table data.